### PR TITLE
Update to version 1.1.3

### PR DESCRIPTION
**Pull request by wenatuhs**
Main new features:

- Adding PV on the fly
- Auto filing in the variable ranges and initial data points table

### CODE REVIEW

File: src/badger/routine.py [hunk 1]
```diff
@@ -150,3 +157,59 @@ def json(self, **kwargs) -> str:
             pass
 
         return json.dumps(dict_result)
+
+
+def calc_bounds(limit_options, vocs, env):
```
Comment (roussel-ryan) [inline src/badger/routine.py:162]: write out calc --> calculate for readability

**Review by roussel-ryan — APPROVED** [review]
mostly cosmetic comments to improve readability down the line. Address feedback and merge

File: src/badger/gui/default/components/routine_runner.py [hunk 2]
```diff
@@ -4,15 +4,17 @@
...
 from pandas import DataFrame
+import torch  # for converting dtype str to torch object
 from PyQt5.QtCore import pyqtSignal, QObject, QTimer
-
-from badger.errors import BadgerRunTerminatedError
+from ....core import run_routine, Routine
+from ....errors import BadgerRunTerminatedError
+from ....tests.utils import get_current_vars
+from ....routine import calc_bounds, calc_init_points
+from ....settings import AUTO_REFRESH
```
[the remark this answers is not in the document]
Comment (wenatuhs) [reply src/badger/gui/default/components/routine_runner.py:15]: Currently it does not belong but we can make it belong -- it's an internal switch that if the new run would go based on current values, or the values stored in that routine. At the moment it's the latter, as we have discussed

File: src/badger/gui/default/components/var_table.py [hunk 3]
```diff
@@ -73,23 +92,27 @@ def header_clicked(self, idx):
...
             sb_upper = self.cellWidget(i, 3)
             self.bounds[name] = [sb_lower.value(), sb_upper.value()]
 
-    def set_bounds(self, variables: dict):
+    def set_bounds(self, variables: dict, signal=True):
         for name in variables:
             self.bounds[name] = variables[name]
 
-        self.update_variables(self.variables, 2)
+        if signal:
```
[the remark this answers is not in the document]
Comment (wenatuhs) [reply src/badger/gui/default/components/var_table.py:105]: It's commented in the `update_variables` method, it's the type of the update -- say, sometimes we need full refresh but other times we only do updates on selected ones, this number tells Badger which way to go

**Pull request merged**